resources/ui-app/package.json

```diff
@@ -11,27 +11,42 @@
   "scripts": {
     "start": "stripes serve",
     "build": "stripes build --output ./output",
-    "test": "stripes test nightmare --run demo --show",
+    "test": "stripes test karma",
     "lint": "eslint ."
   },
   "devDependencies": {
+    "@bigtest/interactor": "^0.8.1",
+    "@bigtest/mirage": "^0.0.1",
+    "@bigtest/mocha": "^0.5.1",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/stripes": "^1.0.0",
     "@folio/stripes-cli": "^1.5.0",
+    "@folio/stripes-core": "^2.15.1",
     "babel-eslint": "^9.0.0",
-    "eslint": "^5.6.0"
+    "babel-polyfill": "^6.26.0",
+    "chai": "^4.2.0",
+    "eslint": "^5.6.0",
+    "mocha": "^5.2.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
+    "react-redux": "^5.0.7",
+    "redux": "^4.0.0",
+    "sinon": "^6.3.4",
+    "webpack": "^4.16.0"
```

Comment:
I believe the CLI is the only package invoking/importing webpack, so it shouldn't be needed within the app template.

Reply:
You're right. I think this was left behind from when mirage was in `.stripesclirc.js`